sequelize/seeders/20180711143600-memeolist-example.js

```diff
@@ -32,14 +32,25 @@ const notesSchema = {
   }
   
   type Subscription {
-    _: Boolean
+    memeAdded(photoUrl: String):Meme!
```

Comment:
[Outside scope of this Pr] I think it will be more benefitial to add more fields to the meme like votes as this will help us to test various use cases.